.github/workflows/gherkin-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Gherkin Linting
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'release/**'
+    # Only run if feature files are changed.
+    paths:
+      - '.github/workflows/gherkin-lint.yml'
```

Comment:
With the package now installed locally, should we also add `package-lock.json` here?

Reply:
Added.